robopi:
  gpio_wheels:
    #from gpio wiring
    left_front_wheel_joint:
      en: 21
      inF: 20
      inB: 16
    left_back_wheel_joint:
      en: 21
      inF: 20
      inB: 16
    right_front_wheel_joint:
      en: 13
      inF: 19
      inB: 26
    right_back_wheel_joint:
      en: 13
      inF: 19
      inB: 26

joint_limits:
  wheel_left:
    has_position_limits: false
    has_velocity_limits: true
    min_velocity: -16.7551608
    max_velocity: 16.7551608
    has_acceleration_limits: false
    min_acceleration: -1.0
    max_acceleration: 1.0
    has_jerk_limits: false
    max_jerk: 0.1
    min_jerk: -0.1
    has_effort_limits: false
    max_effort: 1.0
    min_effort: -1.0
    has_soft_limits: true
    soft_lower_limit: -15.0
    soft_upper_limit: 15.0
    k_position: 10.0
    k_velocity: 10.0
  wheel_right:
    has_position_limits: false
    has_velocity_limits: true
    min_velocity: -16.7551608
    max_velocity: 16.7551608
    has_acceleration_limits: false
    min_acceleration: -1.0
    max_acceleration: 1.0
    has_jerk_limits: false
    max_jerk: 0.1
    min_jerk: -0.1
    has_effort_limits: false
    max_effort: 1.0
    min_effort: -1.0
    has_soft_limits: true
    soft_lower_limit: -15.0
    soft_upper_limit: 15.0
    k_position: 10.0
    k_velocity: 10.0